src/python/pants/backend/python/pip_requirement.py

```diff
@@ -13,14 +13,7 @@
 
 
 class PipRequirement:
-    """A Pip-style requirement.
-
-    Currently just a drop-in replacement for pkg_resources.Requirement.
```

Comment:
Unrelated to the rest of this change, but I noticed that the docstring was out of date.